deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,155 @@
+
+The buildpack's default functionality, as described in this document, is to deploy the OpenTelemetry Collector
+as a sidecar for the given app that's being deployed. The Collector is able to observe the app as a nozzle through
+the Loggregator Firehose. The Loggregator Firehose is one of the architectures Cloud Foundry
+uses to emit logs and metrics. This means that the OpenTelemetry Collector will be observing all
+apps and deployments that emit metrics and logs to the Loggregator Firehose as long as it's running.
+
+## Installation
+- Clone this repository.
+- Change to this directory.
+- Run the following command:
+```sh
+# Add buildpack for OpenTelemetry Collector
+$ cf create-buildpack otel_collector_buildpack . 99 --enable
+```
+
+### Using PCF Buildpack With an Application
+This section covers basic cf CLI (Cloud Foundry Command Line Interface) commands to use the buildpack. 
+```sh
+# Basic setup, see the configuration section for more envvars that can be set
+$ cf set-env <app-name> OTEL_CONFIG <config_file_name>
+$ cf set-env <app-name> OTELCOL <desired_collector_executable_name>
+
+$ cd <my-application-direcory>/
+
+# How to run an application without a manifest.yml file:
+# Note: This will provide the Collector for the app, but the Collector will not be running.
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack>
+
+# How to run an application with a manifest.yml file:
+# Option 1)
+$ cf push
+# Option 2)
+$ cf push <app-name> -b otel_collector_buildpack -b <main_buildpack> -f manifest.yml
+```
+
+Note: This buildpack requires another buildpack to be supplied after it, it is not allowed to
+be the last one for an app. Also, the manifest.yml file will need to provide the
+command to run the OpenTelemetry Collector as a sidecar for the application.
+
+## Configuration
+
+The following only applies if you are using the `otelconfig.yaml` config
+provided by the buildpack.  If you provide a custom configuration file for the OpenTelemetry Collector 
+in your application (and refer to it in the sidecar configuration), these might not
+work unless you have preserved the references to the environment variables in the config file.
+For proper functionality, the `OTEL_CONFIG` environment variable must point to
+the configuration file, whether using the default or a customer version.
+
+Set the following environment variables with `cf set-env` as applicable to configure this buildpack, or
+include them in the `manifest.yml` file, as shown in the [included example](#sidecar-configuration).
+
+Required:
+- `RLP_GATEWAY_ENDPOINT` - The URL of the RLP gateway that acts as the proxy for the firehose,
+    e.g. `https://log-stream.sys.<TAS environment name>.cf-app.com`
+- `UAA_ENDPOINT` - The URL of UAA provider,
+    e.g. `https://uaa.sys.<TAS environment name>.cf-app.com`
+- `UAA_USERNAME` - Name of the UAA user.
+- `UAA_PASSWORD` - Password for the UAA user.
+- `SIGNALFX_ACCESS_TOKEN` - Your SignalFx organization access token.
```

Comment:
I think we've completely moved to the `SPLUNK_` prefix for new offerings. Also SignalFx references should probably be updated to Splunk Observability Suite.

Reply:
Will update. I wasn't aware of the proper terminology, thanks for letting me know!

Reply:
One question, I assume in the actual config file we'll still have the exporter as `signalfx`, right? My assumption is that should be the default exporter for the collector here, but if there's some other endpoint we should be sending this too, let me know.

Reply:
right, the signalfx exporter sends metrics in that format so its component name will likely always be that.

Reply:
Thanks for the clarification! I've removed SignalFx references.